### DISCUSSION TITLE
codespace reload issue

### DISCUSSION THREAD

**Discussion by abbasiahsan399** [community/community#37982] [Codespaces]
Hello, every time I open codespace on my repository, it is showing reload window. i am unable to code on vs code. I keep trying and sometime it is also showing workspace not found. Anyone know solution of this issue mention here so that i can code.
I have good internet connection.
I am using google chrome and used Microsoft edge but issue remain same.
![Screenshot (5)](https://user-images.githubusercontent.com/114071839/199257838-615d53f6-0130-4544-ac11-0d58d0e41db1.png)
 cs50 repository

**Comment by fara-nak** [comment]
Sorry for the issue, would you please open your developer tool (Option + ⌘ + J (on macOS), or Shift + CTRL + J (on Windows/Linux)) and let us see what you see in the console? also can we have your codespace name or id?

**Comment by fara-nak** [comment]
meanwhile have you tried stopping and re-starting your codespace? it might help

**Comment by abbasiahsan399** [comment]
Thank you for response, i am seeing a lot of stuff there(console), i am
unable to understand. i am sending a screenshot here.[image: 114071839
[Codespaces] - Visual Studio Code - Google Chrome 05_11_2022 11_37_50.png]
